lib/checkMergeConflicts.js

```diff
@@ -5,6 +5,10 @@ function sleep(ms) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
+/**
+ * @param {import('probot').Context} context
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ */
 module.exports.checkMergeConflictsInPullRequest = async function(context, pullRequest) {
```

Comment:
Line looks to long, split after bracket maybe.

Reply:
Done.